frontend/scripts/utils/formatValue.js

```diff
@@ -2,7 +2,7 @@ import { NUM_DECIMALS, NUM_DECIMALS_DEFAULT } from 'constants';
 
 // returns a value rounded to numDecimals
 export default (value, dimensionName) => {
-  if (value === undefined || value === null) {
+  if (value === undefined || value === null || value === 'NaN') {
```

Comment:
value being a string `'NaN'` is weird. I think it's a code smell. If anything it should be plain `NaN` and use 'Number.isNaN` to check it.